ml-agents/mlagents/trainers/trainer/trainer_factory.py

```diff
@@ -27,6 +27,7 @@ def __init__(
         init_path: str = None,
         multi_gpu: bool = False,
         force_torch: bool = False,
+        force_tensorflow: bool = False,
```

Comment:
are `force_torch ` and `force_tensorflow` both needed?

Reply:
We left `force_torch` in case users were still using `--torch` in places (discussion here: https://github.com/Unity-Technologies/ml-agents/pull/4582) 

It's behavior is pretty subtle, it will only take effect if the user specifies `framework: tensorflow` in the YAML and then adds `--torch`.